scipy/stats/_continuous_distns.py

```diff
@@ -113,6 +115,66 @@ def _isf(self, q, n):
 ksone = ksone_gen(a=0.0, b=1.0, name='ksone')
 
 
+class kstwo_gen(rv_continuous):
+    r"""General Kolmogorov-Smirnov two-sided test.
```

Comment:
this might be misleading since it is not a test, same in __init__.py

Reply:
Agreed. Changed.